scss/styles.scss

```diff
@@ -9,13 +9,23 @@
 }
 
 .container {
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
   display: flex;
+  padding: 24px;
+  width: 900px;
+}
+
+.book-info {
+  padding: 30px 24px 30px 51px;
+  width: 100%;
```

Comment:
use flex: 1 instead of 100% to avoid bugs with elements without specified width. flex: 1 will make the element grow and take all the available space